src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java

```diff
@@ -21,11 +21,14 @@ public String index() {
         return "index";
     }
 
-    @RequestMapping("/AddFacility.html")
+    @RequestMapping("/addfacility.html")
     public String addFacilityPage() {
-        return "AddFacility";
+        return "addfacility";
```

Comment:
Lowercase is best... however, this did appear in a previous review.  When that occurs, first in wins.